AsyncDisplayKit/Details/ASFlowLayoutController.mm

```diff
@@ -90,10 +90,9 @@ - (NSSet *)indexPathsForScrolling:(ASScrollDirection)scrollDirection rangeMode:(
     currPath.row++;
 
     // Once we reach the end of the section, advance to the next one.  Keep advancing if the next section is zero-sized.
-    while (currPath.row >= [(NSArray *)completedNodes[currPath.section] count] && currPath.section < completedNodes.count - 1) {
+    while (currPath.row >= [(NSArray *)completedNodes[currPath.section] count] && currPath.section < endPath.section) {
```

Comment:
It was a bug, I was able to get it to fail the assertion below. So my rationale was "why have an overbroad condition and then assert against it, rather than setting the condition you want directly?" It serves as a sanity check, I know, but it also seemed silly.